app/client/components/accountoverview/accountOverviewCard.tsx

```diff
@@ -373,7 +398,7 @@ export const AccountOverviewCard = (props: AccountOverviewCardProps) => {
                   )}
                 </li>
               </ul>
-              {!props.productDetail.subscription.payPalEmail && (
+              {!props.productDetail.subscription.payPalEmail && !isGifted && (
```

Comment:
does suggest that some gifted subscriptions come through with `props.productDetail.isPaidTier` = `true` ??

Reply:
or is that the difference between the buyer seeing the gift and the recipient having 'redeemed' the gift?

Reply:
I guess line 65 answers my question 👍